README.md

```
@@ -1,15 +1,20 @@
-DashJ BLS for Android
+# DashJ BLS for Android
 
 ```bash
 git submodule update --init --recursive
 ./gradlew build
 ```
 
-Publish to Maven Local
+### Publish to Maven Local
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)